build.gradle

```diff
@@ -1010,6 +1015,53 @@ project(':core') {
   }
 }
 
+project(':metadata') {
+  archivesBaseName = "kafka-metadata"
+
+  dependencies {
+    compile project(':clients')
+    compile libs.jacksonDatabind
+    compile libs.jacksonJDK8Datatypes
+    compileOnly libs.log4j
+    testCompile libs.junitJupiter
+    testCompile libs.hamcrest
+    testCompile libs.slf4jlog4j
+    testCompile project(':clients').sourceSets.test.output // for org.apache.kafka.test.IntegrationTest
```

Comment:
I think you don't need this if that comment is true.

Reply:
later PRs add IntegrationTest to the metadata module... simpler to just keep it in this PR

Reply:
This is not right, the mentioned annotation is only used for JUnit 4. I submitted a minor PR that fixes the comment at least: https://github.com/apache/kafka/pull/9897